docsrc/index.nim

```diff
@@ -132,7 +132,7 @@ Currently most of the documentation on customization is given by the examples.
 
 ### other blocks
 
-You can find a complete description of the code blocks along with examples in [allblocks.nim](https://pietroppeter.github.io/nimib/allblocks.html)
+You can find a complete description of the code blocks along with examples in [allblocks]({docs}/allblocks.html.
```

Comment:
You accidentally deleted the ending `)` for this link.

Reply:
```suggestion
You can find a complete description of the code blocks along with examples in [allblocks]({docs}/allblocks.html).
```